src/test/resources/features/createContract.feature

```diff
@@ -27,3 +27,158 @@ Feature: Create Contract
+      | Owner Expiration Notice | 30 Days             |
+      | Company Signed Date     | 7/19/2021           |
+      | Special Terms           | Special Terms       |
+      | Description             | Description         |
+    Then I verify Contract created with all fields
+      And I verify that the date matches the creation date
+
+  @CreateContract
+  Scenario: create a contract with Account Name and price book as pre-postcondition
+    Given I navigate to the "Contract" page
+    When I create a new contract with all fields
+      | Account Name            | TestAccount         |
+      | Contract Term (months)  | 2                   |
+      | Contract Start Date     | 8/15/2021           |
+      | Customer Signed Date    | 7/15/2021           |
+      | Price Book              | Standard            |
+      | Owner Expiration Notice | 30 Days             |
+      | Company Signed Date     | 7/19/2021           |
+      | Special Terms           | Special Terms       |
+      | Description             | Description         |
+    Then I verify Contract created with all fields
+      And I verify that the date matches the creation date
+
+  @CreateContract
+  Scenario: create a contract with Account Name and price book as pre-postcondition
+    Given I navigate to the "Contract" page
+    When I create a new contract with all fields
+      | Account Name            | TestAccount         |
+      | Contract Term (months)  | 2                   |
+      | Contract Start Date     | 8/15/2021           |
+      | Price Book              | Standard            |
+      | Owner Expiration Notice | 30 Days             |
+      | Company Signed Date     | 7/19/2021           |
+    Then I verify Contract created with all fields
+      And I verify that the date matches the creation date
+
+  @CreateContract
+  Scenario: create a contract with Account Name and price book as pre-postcondition
+    Given I navigate to the "Contract" page
+    When I create a new contract with all fields
+      | Account Name            | TestAccount         |
+      | Contract Term (months)  | 2                   |
+      | Contract Start Date     | 8/15/2021           |
+      | Price Book              | Standard            |
+      | Company Signed Date     | 7/19/2021           |
+    Then I verify Contract created with all fields
+      And I verify that the date matches the creation date
+
+  @CreateContract
+  Scenario: create a contract with Account Name and price book as pre-postcondition
+    Given I navigate to the "Contract" page
+    When I create a new contract with all fields
+      | Account Name            | TestAccount         |
+      | Contract Term (months)  | 2                   |
+      | Contract Start Date     | 8/15/2021           |
+      | Price Book              | Standard            |
+      | Owner Expiration Notice | 30 Days             |
+    Then I verify Contract created with all fields
```

Comment:
Could be more descriptive like I  verify that the created Contract contains the correct information.

Reply:
Ok, It was solved, thanks for the suggestion